x-pack/plugins/viz_editor/public/common/components/visualization_modal/visualization_modal.scss

```diff
@@ -0,0 +1,11 @@
+.visualizationModal {
```

Comment:
Names aren't really prefixed. Not an issue today, but will be an issue for production